tools/accuracy_checker/accuracy_checker/annotation_converters/ms_coco.py

```diff
@@ -105,7 +105,11 @@ def convert(self, check_content=False, progress_callback=None, progress_interval
         image_ids = [(image['id'], image['file_name'], np.array([image['height'], image['width'], 3]))
                      for image in image_info]
         if self.sort_annotations:
-            image_ids.sort(key=lambda value: value[0])
+            if self.sort_key == 'image_id':
+                image_ids.sort(key=lambda value: value[0])
+            else:
+                image_ids.sort(key=lambda value: tuple(value[2]))
```

Comment:
on my view better to have dict with lambda functions as values and key is sort_key parameters. It will be easy extend in future if we will need to add any other sort keys

Reply:
Done